src/components/scenes/GuiPluginViewScene.tsx

```diff
@@ -190,7 +190,7 @@ export function GuiPluginViewScene(props: Props): JSX.Element {
       : 'Mozilla/5.0 (iPhone; CPU iPhone OS 10_3_1 like Mac OS X) AppleWebKit/603.1.30 (KHTML, like Gecko) Version/10.0 Mobile/14E304 Safari/602.1'
 
   return (
-    <SceneWrapper background="theme">
+    <SceneWrapper background="theme" hasTabs={route.name !== 'pluginView'}>
```

Comment:
Didn't I do this for this scene from a previous review? I was confident that I added this as a fixup from review feedback. What happened there in retrospect, I wonder.

Reply:
Yes, you did do it. However, it got lost during the EdgeProvider re-write.